packages/trader/src/App/Components/Elements/Calendar/calendar-header.jsx

```diff
@@ -64,6 +64,10 @@ const CalendarHeader = ({
         () => navigateTo(addMonths(calendar_date, 1)) :
         undefined;
 
+    const monthButtonOnClick = () => disable_month_selector ? undefined : switchView('month');
+    const yearButtonOnClick = () => is_select_year_disabled ? undefined : switchView('year');
+    const decadeButtonOnClick = is_select_year_disabled ? undefined : () => switchView('decade');
```

Comment:
nit: could you make this a function as above: `const decadeButtonOnClick = () => {}`?